.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+- id: zizmor
+  name: zizmor
+  description: 'Find security issues in GitHub Actions CI/CD setups'
+  language: rust
+  files: \.github/workflows/.*\.yml$
+  entry: zizmor
```

Comment:
Out of curiosity: how does `pre-commit` know to build + run `zizmor` here? Does it recognize that the repo contains a crate and then build it locally?

(I don't use pre-commit hooks much, so sorry if this is a silly question.)

Reply:
Not a silly question! `language: rust` tells pre-commit what to do.
Refs : https://pre-commit.com/index.html#rust

Reply:
That's very cool!